<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Navigation</class>
 <widget class="QWidget" name="Navigation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,3">
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>飞行信息</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_flightHeight">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="maximum">
            <double>9999.989999999999782</double>
           </property>
           <property name="value">
            <double>472.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>飞行高度(m):</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>目标值</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>飞行速度(km/h):</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string>+3.12</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 255);
</string>
           </property>
           <property name="text">
            <string>-10.13</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>偏差</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_flightSpeed">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <double>9999.989999999999782</double>
           </property>
           <property name="value">
            <double>157.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>航向角(°)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="spinBox">
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>360</number>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_8">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 255);</string>
           </property>
           <property name="text">
            <string>-0.8265</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeWidget_navigation">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string>参数项</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>值</string>
         </property>
        </column>
        <item>
         <property name="text">
          <string>系统参数</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <item>
          <property name="text">
           <string>当前飞行速度</string>
          </property>
          <property name="text">
           <string>146.77</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>测区覆盖率</string>
          </property>
          <property name="text">
           <string>0.4318</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>当前航线</string>
          </property>
          <property name="text">
           <string>10</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>GPS信息</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <item>
          <property name="text">
           <string>周</string>
          </property>
          <property name="text">
           <string>2123</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>秒</string>
          </property>
          <property name="text">
           <string>465820.00</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>细分时间(s)</string>
          </property>
          <property name="text">
           <string>0.159813</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>纬度(°)</string>
          </property>
          <property name="text">
           <string>32.416978</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>经度(°)</string>
          </property>
          <property name="text">
           <string>119.513329</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>高度(m)</string>
          </property>
          <property name="text">
           <string>381.298</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>翻滚角(°)</string>
          </property>
          <property name="text">
           <string>0.25937</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>俯仰角(°)</string>
          </property>
          <property name="text">
           <string>2.85019</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>航向角(°)</string>
          </property>
          <property name="text">
           <string>359.17349</string>
          </property>
         </item>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QPushButton" name="btn_loadTracker">
           <property name="text">
            <string>加载航线</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="lineEdit_navigationFile">
           <property name="text">
            <string>2018_03_07_20_23_50_833.datagps</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,5,1">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Courier New&quot;;</string>
        </property>
        <property name="text">
         <string>飞行高度：+3.12m      飞行速度：-10.13km/h     航向角：-0.8265°</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_info">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCustomPlot" name="navigationPlot" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_test">
        <property name="title">
         <string>GroupBox</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>CheckBox</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">./src/third_lib/QCustomPlot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
